.github/workflows/ci.yml

```diff
@@ -138,44 +144,83 @@ jobs:
         run: ./gradlew sonar --parallel --build-cache
 
 
-      # Assemble test
-      - name: Assemble tests
-        if: always()
-        id: compile-tests
-        run: |
-          ./gradlew compileDebugAndroidTestKotlin --parallel --build-cache
-
+      - name: Create empty result file
+        run: echo -n "" > grading-result.csv
 
       # This step retrieve all useful data from the tests
       - name: Retrieve results
         run: |
-          sudo apt-get install -y libxml2-utils 
+          # Tasks results
+          echo "build-task,${{ steps.build.outcome }}" >> grading-result.csv
+          echo "unit-tests-task,${{ steps.unit-tests.outcome }}" >> grading-result.csv
+          echo "compile-tests,${{ steps.compile-tests.outcome }}" >> grading-result.csv
+          echo "instrumentation-tests-task,${{ steps.instrumentation-tests.outcome }}" >> grading-result.csv
+          echo "coverage-task,${{ steps.coverage.outcome }}" >> grading-result.csv
+          
+          # Tests results
+          
+          result_file="app/build/outputs/androidTest-results/connected/*/TEST-*.xml"
+          
+          test_names=("useAppContext")
+          test_classes=("com.github.se.bootcamp.todo.ExampleInstrumentedTest")
```

Comment:
Do you only test ExampleInstrumentedTest or start from this one?

Reply:
I used this to explicitly check if this test is run, to fetch the results. But normally all the tests in androidTest should be run